### PR TITLE
New Article: Firewall manager v2

### PR DESCRIPTION
**Pull request by trevorbecker**
Speaking with Nate now
sending email now with 2 images
This is first draft of my fwcpv2 landing page

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b66a7fa22a/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-addaf9e038/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d2f7e64add/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3f0101ab7c/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9ea166849d/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-d19c976adb/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-331418b6bb/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2f26b281fa/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1c6c163928/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Here are my suggested edits for the article. Please let me know if you have any questions. Thanks! 

[Firewall Manager v2_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/873268/Firewall.Manager.v2_editsKH.pdf)
[Firewall Manager v2_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/873269/Firewall.Manager.v2_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5fa05eb251/how-to/). :bow:

File: content/dedicated-hosting/firewall-manager-v2-beta.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+last_modified_by: Nate Archer
+product: Dedicated Hosting
+product_url: dedicated-hosting
+---
+
+Firewall Manager v2 is a new tool within the MyRackspace portal. This article describes the tool and provides links to detailed articles about the Firewall Manager v2 features.
+
+### What is Firewall Manager?
+
+Firewall Manager, or also commonly referred to as the Firewall Control Panel (FWCP), is a self-service tool in the MyRackspace portal. Firewall Manager enables you to immediately view and modify the configurations of your Cisco firewalls by securely connecting to them and executing the changes on your behalf. In many cases, you can use Firewall Manager to execute a change in less time than it takes to create a ticket.
```
Comment (catlook) [inline content/dedicated-hosting/firewall-manager-v2-beta.md]: I'd say "or commonly", "also commonly", or just "commonly" but not both.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-054fc401e5/how-to/)

**Comment by DonNateR** [comment]
@trevorbecker: I'm going to loop back around and add the images for this one once I get the other articles merged. This way, I can prevent any potential merge conflicts that may occur.

**Pull request merged**